plugins/ocm-backend/dev/types.ts

```diff
@@ -10,10 +10,10 @@ import { Config } from '@backstage/config';
 import { IdentityApi } from '@backstage/plugin-auth-node';
 import { PermissionEvaluator } from '@backstage/plugin-permission-common';
 
-import { Logger } from 'winston';
+import { LoggerService } from '@backstage/backend-plugin-api';
 
```

Comment:
For the OCM backend plugin, you need to do the following modifications:
Remove this import:
https://github.com/janus-idp/backstage-plugins/blob/835e63f3995fb840108118681f6e012f7ac88f08/plugins/ocm-backend/src/providers/module.ts#L17
Change remove the wrapper and just pass `logger,`:
https://github.com/janus-idp/backstage-plugins/blob/835e63f3995fb840108118681f6e012f7ac88f08/plugins/ocm-backend/src/providers/module.ts#L40
Change all `winston.Logger` instances to `LoggerService` in https://github.com/janus-idp/backstage-plugins/blob/835e63f3995fb840108118681f6e012f7ac88f08/plugins/ocm-backend/src/providers/ManagedClusterProvider.ts

Reply:
@Zaperex I've made these changes